### ISSUE TITLE
`uninitialized constant XPath::HTML` error when using `When I fill in "..." with "..." inside any "..."`

### ISSUE THREAD

**Issue by danielsmakandra** [makandra/spreewald#210]
I found a bug in a step definition:

I tried to use the `When I fill in "..." with "..." inside any "..."` step but got the following error message:
```
uninitialized constant XPath::HTML

    input = container.first(:xpath, XPath::HTML.fillable_field(field))
                                         ^^^^^^ (NameError)
features/movies/search_cucumber.feature:16:in `I fill in "Search movies" with "#{query}" inside any ".searchable"'
```
The [step's code](https://github.com/makandra/spreewald/blob/eee25428b99dd71bb32590eee08cbac12b8d1934/lib/spreewald/web_steps.rb#L619-L630) seems to use `XPath::HTML` from the [xpath gem](https://rubygems.org/gems/xpath).

[`Xpath::HTML` was removed in xpath >= 3.0.0](https://github.com/teamcapybara/xpath/commit/d18fe71be628fff8426798acbe56fee03862be8e) and thus the step fails.

**Comment by FLeinzi** [comment]
Hi @danielsmakandra,

thanks for the hint. It's definitely broken. I will rewrite this step soon.

The `"..." within "..."` step works similar to the one above, but will only search inside the first match. So you may use a more specific selector.

Like this: `I fill in "Search movies" with "#{query}" within ".parent .searchable"`

**Comment by FLeinzi** [comment]
I released a fix as version `4.6.2`.
